pkg/review/commit_review_status.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package review contains code to get review status information
+// for a GitHub commit.
+
+package review
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/shurcooL/githubv4"
+)
+
+func GetPullRequests(client *githubv4.Client, githubOrg, repository, commitSha string) ([]PullRequest, error) {
```

Comment:
Can we return a slice of pointers here? `[]*PullRequest`

Reply:
done.

Reply:
Is there a reason for preferring a slice of pointers vs. a slice of structs. My understanding is that a Go Slice is just a header that points to the underlying array, so we shouldn't incur any copying cost by returning a slice of structs.

Also a slice of structs should have better cache locality and fewer memory allocations as we can just allocate one contiguous block of memory, whereas if we did a slice of pointers there would need to be separate allocations for each individual struct.

Reply:
This is more about convention than performance. I tend to use pointers whenever we can.

Reply:
Yea, I should have clarified, referring to our teams style guide: https://github.com/abcxyz/readability/wiki/Go-Best-Practices#pointers

Reply:
Ok, I've updated it to return `[]*PullRequest`